### DISCUSSION TITLE
problem accessing the student starter pack

### DISCUSSION THREAD

**Discussion by MaxenceMnz** [community/community#69486] [GitHub Education]
### Hi everyone,

Hello, I have a little problem. I'm currently at IT school and I'd like to access the starter pack for students. The only problem is that when I enter the information, I always get the same message (see attachment) even though I put my first name / surname in the tab that the site asks me to correct. Thank you for your help

**Comment by mantasu** [comment]
You need to have your first name and last name on your GitHub profile too

**Reply by MaxenceMnz** [reply]
in profil or in "Billing and plans" ? because I've already entered it in the biling and plans section with my address etc.

**Reply by mantasu** [reply]
Profile as well, see the answer [here](https://github.com/orgs/community/discussions/68725#discussioncomment-7163466). You can always change it back after verification, but it's easier for the staff to verify I presume

**Reply by MaxenceMnz** [reply]
I still have the same problem with my name, even if I put the English translation next to it, it still doesn't work.
